backend/chainlit/discord/__init__.py

```diff
@@ -1,6 +1,6 @@
-try:
-    import discord
-except ModuleNotFoundError:
+import importlib.util
+
+if importlib.util.find_spec("discord") is None:
     raise ValueError(
         "The discord package is required to integrate Chainlit with a Slack app. Run `pip install discord --upgrade`"
```

Comment:
Unrelated, but did someone forget to change Slack to Discord here? @willydouhard

Reply:
Yes, unrelated, but fixed in c9aaa4b9d247a4435056495ae1d86ea6f565f180.